pkg/v1/sdk/capabilities/hack/generate-package-secret.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
nit: can we move all these files under `pkg/v1/sdk/capabilities/hack/package/generate-package-secret.sh` just so that it is more segregated?